index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="./css/landing.css">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>MY BRAND</title>
+</head>
+<body>
+<nav class="fixed-nav-bar" id="navbar">
+    <div class="logo">
+        <h3><a href="#">MR CHRIS</a></h3>
+    </div>
+    <ul class="nav-links">
+        <li><a href="#">Home</a></li>
```

Comment:
Good job @Mucyo-Aime-Christian, kindly remember to do the actual link between different sections, for instance when you click on blog in the nav you should go to the blog section and so on...